docs: add Sep 2024 meeting minutes (#97)
oasis-tcs/openeox: meeting-minutes/2024/2024-09-18.md

# 1. Opening Activities

## 1.1 Opening comments (Co-Chair)

- Justin & Omar open the meeting.

## 1.2 Introduction of participants/roll call (Co-Chair)

> Quorum requires participation of 8 or more of the 14 voting members (including the officers).

Reviewer: ```suggestion
Quorum requires participation of 8 or more of the 15 voting members (including the officers).
```

| First Name | Last Name   | Company                                                     | Role(s)       | Present |
|:-----------|:------------|:------------------------------------------------------------|:--------------|:--------|
| Adrian     | Diglio      | Microsoft                                                   | Member        | No      |

> | Art        | Middlekauff | Flexera                                                     | Voting Member | No      |

Reviewer: Art did not join the TC until January 2025
```suggestion
```

| David      | Kemp        | National Security Agency                                    | Member        | No      |
| Denny      | Page        | Individual                                                  | Voting Member | Yes     |
| Feng       | Cao         | Oracle                                                      | Voting Member | Yes     |
| Harin      | Sarda       | Cisco Systems                                               | Voting Member | Yes     |
| Jautau     | White       | Microsoft                                                   | Voting Member | No      |
| Jeremy     | Rickard     | Microsoft                                                   | Member        | No      |
| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Co-Chair      | Yes     |

> | Kris       | Vandecruys  | Cisco Systems                                               | Member        | Yes     |

Reviewer: Kris was attending regularly and Voting Member at this time.
```suggestion
| Kris       | Vandecruys  | Cisco Systems                                               |  Voting Member  | Yes     |
```

| Kunal      | Modasiya    | Qualys, Inc.                                                | Member        | No      |

> | Langley    | Rock        | Dell                                                        | Voting Member | Yes     |

Reviewer: Looking at [July and August Meeting Minutes](meeting-minutes/2024), it appears that Langley would have lost Voting Members status for this meeting.
```suggestion
| Langley    | Rock        | Dell                                                        | Member       | Yes     |
```

> | Martin     | Prpic       | Red Hat                                                     | Member        | No      |

Reviewer: Tracking that Martin had voting status at this time. Should have lost Voting status after October meeting.
```suggestion
| Martin     | Prpic       | Red Hat                                                     | Voting Member        | No      |
```

| Omar       | Santos      | Cisco Systems                                               | Co-Chair      | Yes     |

> | Pablo      | Quiroga     | Qualys, Inc.                                                | Member        | Yes     |

Reviewer: Pablo had voting member status at this time.
```suggestion
| Pablo      | Quiroga     | Qualys, Inc.                                                | Voting Member | Yes     |
```

| Peter      | Gephardt    | IBM                                                         | Member        | No      |
| Przemyslaw | Roguski     | Red Hat                                                     | Voting Member | Yes     |

> | Sergii     | Demianchuk  | Cisco Systems                                               | Member        | No      |

Reviewer: Sergii did not start attending meetings until 2025.
```suggestion
```

## 1.3 Procedures for this meeting (Moderator)

## 1.4 Approval of agenda

- Roll Call  
- Updates
- Approval of previous meeting minutes (motions carried out per e-mail motions)  
- Review of actions
- Review of outstanding issues and pull requests marked for TC discussion: [https://github.com/oasis-tcs/openeox](https://github.com/oasis-tcs/openeox)  
- Next steps  

## 1.5 Approval of previous minutes (Moderator)

- Motions will be set via mailing list.

## 1.6 Review of action items and resolutions (Secretary Stefan)

## 1.7 Identification of TC voting members (Secretary)

### 1.7.1 Prospective voting members attending their first meeting

### 1.7.2 Members attaining voting rights at the end of this meeting

### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends

### 1.7.4 Members who previously lost voting rights who are attending this meeting

### 1.7.5 Members who have declared a leave of absence

# 2. Future Meetings

## 2.1 Future meeting schedule (Secretary)

- Next Scheduled Teleconferences (Wednesdays at 09:00 PT / 12:00 ET / 18:00 CET / **17:00** UTC for 1 hour)
```
October 16, 2024
```

# 3. Discussion

## 3.1 Updates

> - Justin received positive feedback on his presentation on OpenEoX at the open source security conference in Vienna.

Reviewer: ```suggestion
- Justin received positive feedback on his presentation on OpenEoX at the Open Source Security Summit Europe in Vienna.
```

> Open EOX was highlighted as one of the activities in OASIS during the all hands meeting.

Reviewer: ```suggestion
OpenEoX was highlighted as one of the activities in OASIS during the all hands meeting.
```

## 3.2 Issues / Pull Requests

- The TC discussed the need to organize and improve the value scenarios, with folders for Red Hat, Oracle, and Dell.
- The vendor definition and product life cycle definitions were discussed and may need some revisions.
- The team discussed the need to include version information in the product life cycle, as it is important for clarity and usefulness.
- Sonny suggested referencing an ISO standard for additional information on support processes, but noted that it may not be necessary.
- There was a discussion about whether to use the term "end of maintenance" or "end of support," with some members advocating for clarity and consistency in terminology.
- The group discussed the need for clear definitions and terms regarding hardware support and maintenance.
- There is a debate about whether the standard should include security aspects and vulnerability support.
- The group agreed that there should be standardized scopes for end of maintenance, but the specific definitions need to be determined.

> - [Pull request #40](https://github.com/oasis-tcs/openeox/pull/40) was created to add items and instructions for contributors to the repository.

Reviewer: ```suggestion
- [Pull request #40](https://github.com/oasis-tcs/openeox/pull/40) was created for TC members to add their use cases and value scenarios.
```
